java/client/src/org/openqa/selenium/remote/ErrorHandler.java

```diff
@@ -220,7 +220,7 @@ private String duration(long duration) {
 
   private Throwable rebuildServerError(Map<String, Object> rawErrorData, int responseStatus) {
 
-    if (!rawErrorData.containsKey(CLASS) && !rawErrorData.containsKey(STACK_TRACE)) {
+    if (!(rawErrorData.containsKey(CLASS) && rawErrorData.get(CLASS) != null) && !(rawErrorData.containsKey(STACK_TRACE) && rawErrorData.get(STACK_TRACE) != null)) {
```

Comment:
`!(rawErrorData.containsKey(CLASS) && rawErrorData.get(CLASS) != null)`
can be simplified to 
`rawErrorData.get(CLASS) == null`
and the same for the second condition.